Dockerfile.ubuntu

```diff
@@ -36,7 +38,7 @@ ENV HDDL_INSTALL_DIR=/opt/intel/openvino/deployment_tools/inference_engine/exter
 ENV DEBIAN_FRONTEND=noninteractive
 ENV TF_SYSTEM_LIBS="curl"
 SHELL ["/bin/bash", "-c"]
-RUN apt update && apt install -y \
+RUN apt-get update && apt-get install -y \
```

Comment:
'apt' should suffice

Reply:
apt-get is actually preferred in dockerfiles. it will be replaces later in all instances as part of hadolint check